props/prop_base.h

```diff
@@ -293,6 +297,36 @@ class PropBase : CommandParser, Looper, protected SaberBase {
 #endif    
   }
 
+  void ResumePreset() {
+    FileReader f;
+    int newPreset;
+    int newVolume;
+    if (f.Open("savedpreset.ini") && f.FileSize() > 0) {
```

Comment:
The code that reads the preset file and the code that writes the preset file should be in the same file.

Reply:
moved savestate function to prop_base.h